src/main/java/com/google/sps/data/UrlRequest.java

```diff
@@ -0,0 +1,68 @@
+package com.google.sps.data;
+
+import java.io.IOException;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.UnsupportedEncodingException;
+import java.nio.charset.StandardCharsets;
+import java.net.URLEncoder;
+import java.net.URL;
+import java.net.HttpURLConnection;
+import java.util.Map;
+
+/* Class conatining helper methods to make url requests */
+public final class UrlRequest {
```

Comment:
More a question then a comment, but why did you choose to make another java class outside of the servlet?

Reply:
On second look, this makes a lot of sense. I'll probably incorporate using this class into my servlets in a later PR.